### ISSUE TITLE
Option to switch teams before starting the game

### ISSUE THREAD

**Issue by westonkelliher** [westonkelliher/KeyWords#58]

**Comment by safwaan-menk** [comment]
Duplicate of #52

**Comment by westonkelliher** [comment]
Perhaps the way to do this is keep players on the team selection screen after they've selected a role and only once the host decides to star the game does it start
